code/modules/mob/living/simple_animal/hostile/megafauna/demonic_frost_miner.dm

```diff
@@ -51,6 +54,8 @@ Difficulty: Extremely Hard
 
 /mob/living/simple_animal/hostile/megafauna/demonic_frost_miner/Initialize()
 	. = ..()
+	for(var/obj/structure/frost_miner_prism/P in GLOB.frost_miner_prisms)
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names